src/ingest_validation_tools/schema_loader.py

```diff
@@ -260,6 +260,19 @@ def get_directory_schema(
     return schema
 
 
+def get_possible_directory_schemas(dir_schema: str) -> Optional[Dict]:
+    schemas = {}
+    # this assumes that versions are numbered starting at x.0, no whole numbers
+    directory_schema_minor_versions = _directory_schemas_path.glob(f"{dir_schema}*.yaml")
```

Comment:
If dir_schema is 'codex-v1', won't that wildcard match 'codex-v17' ?  That is, don't you need an explicit '.' ?